pkg/remotewrite/config.go

```diff
@@ -229,6 +255,14 @@ func parseEnvs(env map[string]string) (Config, error) {
 		c.Password = null.StringFrom(password)
 	}
 
+	if clientCertificate, certDefined := env["K6_PROMETHEUS_RW_CLIENT_CERTIFICATE"]; certDefined {
```

Comment:
```suggestion
	if clientCertificate, certDefined := env["K6_PROMETHEUS_RW_TLS_CLIENT_CERTIFICATE"]; certDefined {
```
Do you think we should be more specific about the naming?

Reply:
Both fine fro me :shrug:

Reply:
Is there something else that we can have a client certificate for ?

If not I am also :shrug: - if yes , then I would prefer to have `mtls` even.

Reply:
We are doing mostly the same with Username and Password where we aren't specifying that it is for BasicAuth.